lib/modules/manager/docker-compose/schema.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from 'zod';
+
+const DockerComposeService = z.object({
+  image: z.string().optional(),
+  build: z
```

Comment:
@secustor `build` does not need to be an object in the compose spec, it may be, and often is, a string.

https://docs.docker.com/compose/compose-file/compose-file-v2/#build

This has broken a bunch of our compose files that use the shorthand `build: ./path` and compose files can no longer be parsed.